Jenkinsfile

```diff
@@ -0,0 +1,47 @@
+pipeline {
+  agent { label 'linux' }
+
+  options {
+    disableConcurrentBuilds()
+    /* manage how many builds we keep */
+    buildDiscarder(logRotator(
+      numToKeepStr: '20',
+      daysToKeepStr: '30',
+    ))
+  }
+
+  environment {
+    GIT_AUTHOR_NAME = 'status-im-auto'
+    GIT_AUTHOR_EMAIL = 'auto@status.im'
+    /* Dev site deployment. */
+    DEV_SITE = 'dev.vac.dev'
```

Comment:
Huh? Status specs should be separate from Vac.dev

Reply:
Typo maybe?

Reply:
Oh, that's just a leftover because I copied the `Jenkinsfile`. There is no `dev` in this case since this repo doesn't have a `develop` branch, I'll remove that.